xlive/Blam/Engine/rasterizer/dx9/rasterizer_dx9_screen_effect.cpp

```diff
@@ -40,8 +40,6 @@ void __cdecl rasterizer_dx9_render_crossfade(real32 lower, real32 upper);
 
 void rasterizer_dx9_screen_effect_apply_patches(void)
 {
-	PatchCall(Memory::GetAddress(0x191BB1), rasterizer_dx9_postprocess_scene);
```

Comment:
was this a intended patch removal?

Reply:
yes, it's leftover from when the caller wasn't rewritten